{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/utilities.py

```diff
@@ -77,3 +79,74 @@ def read_data_by_draw(artifact_path: str, key : str, draw: int) -> pd.DataFrame:
     data = pivot_categorical(data)
     data[project_globals.LBWSG_MISSING_CATEGORY.CAT] = project_globals.LBWSG_MISSING_CATEGORY.EXPOSURE
     return data
+
+
+def get_norm(
```

Comment:
if you get the 95% interval, you can derive the mean since the distribution is symmetric. Probably keep it required so the interface is consistent, but you should definitely check that the provided mean is the midpoint of the provided interval.

Reply:
It's actually pretty common that the mean is not the exact midpoint of the confidence interval.